### ISSUE TITLE
TAG Issue, likley unique to Canada (AIP Listed)

### ISSUE THREAD

**Issue by sbuchanYVR** [Helipaddy/report#2]
- ‘AIP listed’ should be ‘AIP/CFS listed’ for Canada, ‘AIP Listed’ for the US
most Canadian Pilots won't know what the AIP is, as we use the term CFS,

**Comment by danielacojocari** [comment]
@paddywills  can we have it changed to 'AIP/CFS listed'? To have lists per user's country is additional logic, seems extra for only one item for now

**Comment by sbuchanYVR** [comment]
Of course.  That worksOn Oct 8, 2023, at 11:46 PM, danielacojocari ***@***.***> wrote:﻿
@paddywills  can we have it changed to 'AIP/CFS listed'? To have lists per user's country is additional logic, seems extra for only one item for now
